File: LOO 2°Parcial/Examen2/src/Clases/Conica.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package Clases;
import Clases.Point2D;
/**
 * 
 * @author Dimitri Medina Novelo
 */
public abstract class Conica {
    public static int numObj = 0;
    public abstract boolean estaSobre(Point2D obj);
    public abstract Point2D mover(float _x, float _y);
    public abstract String ecuacionCanonica();
    public abstract String toString();
}

File: LOO 2°Parcial/Medina Novelo Jesus/py/Conica.py
from abc import ABC, abstractmethod

class Conica(ABC):

    @abstractmethod
    def estaSobre(self):
        pass

    @abstractmethod
    def mover(self):
        pass

    @abstractmethod
    def ecuacionConica(self):
        pass

    @abstractmethod
    def toString(self):
        pass